packages/lesswrong/server/migrations/meta/utils.ts

```diff
@@ -174,7 +175,7 @@ export const updateIndexes = async <N extends CollectionNameString>(
   collection: CollectionBase<N>,
 ): Promise<void> => {
   for (const index of expectedIndexes[collection.collectionName] ?? []) {
-    collection._ensureIndex(index.key, index);
+    await collection._ensureIndex(index.key, index);
```

Comment:
This was not being awaited before, I'm not sure what the behaviour would have been wrt what happens when the transaction closes

Reply:
It's because the type was `any` before, so no need to change anything with eslint